xtask/src/kotlin.rs

```diff
@@ -21,14 +20,8 @@ enum Package {
 impl Package {
     fn values(self) -> PackageValues {
         match self {
-            Package::CryptoSDK => PackageValues {
-                name: "matrix-sdk-crypto-ffi",
-                udl_path: "bindings/matrix-sdk-crypto-ffi/src/olm.udl",
-            },
-            Package::FullSDK => PackageValues {
-                name: "matrix-sdk-ffi",
-                udl_path: "bindings/matrix-sdk-ffi/src/api.udl",
-            },
+            Package::CryptoSDK => PackageValues { name: "matrix-sdk-crypto-ffi" },
```

Comment:
I only noticed this task also builds the CryptoSDK following clippy warnings. This change is untested for that product.